bigbluebutton-html5/imports/ui/components/audio/audio-controls/container.jsx

```diff
@@ -38,6 +41,10 @@ const processToggleMuteFromOutside = (e) => {
 };
 
 const handleLeaveAudio = () => {
+  if (getFromUserSettings('bbb_skip_check_audio_on_first_join', APP_CONFIG.skipCheckOnJoin)) {
```

Comment:
Is there a specific reason you placed this code in the `handleLeaveAudio`?

Reply:
the `handleLeaveAudio` needs to know if we are only skipping the echo on the first join so it can set the flag to override the skipping